packages/devtools/src/commands/executeAsyncScript.js

```diff
@@ -5,6 +5,16 @@ import { SERIALIZE_PROPERTY, SERIALIZE_FLAG } from '../constants'
 export default async function executeAsyncScript ({ script, args }) {
     const page = this.getPageHandle(true)
     const scriptTimeout = this.timeouts.get('script')
+    script = script.trim()
+
+    if (script.startsWith('return (')) {
```

Comment:
Do we have test for this one?

Reply:
yes, see in `e2e/protocol.test.js`
```js
    it('can handle various of script formats', async () => {
        expect(await browser.executeScript('console.log("hello")')).toBe(undefined)
        expect(await browser.executeScript(' return "string";')).toBe('string')
        expect(await browser.executeScript('/* test */ console.log("test")')).toBe(undefined)
        expect(await browser.executeScript('return { foo: "bar" }')).toEqual({ foo: 'bar' })
    })
```

Reply:
we test all commands currently like that as it just makes more sense to have one common e2e test.

Reply:
Should we have there script that starts with `return (`?

Reply:
added such a test

Reply:
thanks